src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1179,11 +1180,18 @@ public override void WriteDebugLine(string message)
             }
             else
             {
-                // NTRAID#Windows OS Bugs-1061752-2004/12/15-sburns should read a skin setting here...
-                WriteLine(
-                    DebugForegroundColor,
-                    DebugBackgroundColor,
-                    StringUtil.Format(ConsoleHostUserInterfaceStrings.DebugFormatString, message));
+                if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
+                {
+                    message = Utils.GetFormatStyleString(Utils.FormatStyle.Debug) + Utils.GetOutputString(message, isHost: true, SupportsVirtualTerminal, Console.IsOutputRedirected);
```

Comment:
`WriteLine` will eventually call `WriteImpl`, where you call `Utils.GetOutputString`, so maybe you don't need to call `Utils.GetOutputString` here?

Reply:
yeah, looks like they all end up at WriteImpl, will update